generic/tests/pktgen_perf.py

```diff
@@ -109,10 +110,9 @@ def install_package(ver, session=None):
         output_cmd(cmd_install)
         output_cmd(cmd_clean)
 
-    check_cmd = "uname -r |grep el8"
-    if process.run(check_cmd, shell=True):
+    if int(re.findall('el(\\d)', process.getoutput('uname -r'))[0]) > 7:
         install_package(host_ver)
-    if session.cmd(check_cmd):
+    if int(re.findall('el(\\d)', session.cmd('uname -r'))[0]) > 7:
```

Comment:
Sorry, I have missed the condition that `el` does not exist, which might lead to out of index error, maybe we could modify it like this:
```
def is_version_lt_rhel7(uname_str):
    ver = re.findall('el(\\d)', uname_str)
    if ver:
        return int(ver[0]) > 7
    return False

if is_version_lt_rhel7(process.getoutput('uname -r')):
    install...
if is_version_lt_rhel7(session.cmd('uname -r')):
    ...
```

Reply:
I have updated. 

JOB LOG    : /root/avocado/job-results/job-2021-03-14T23.41-78dd146/job.log
 (1/2) 82599.bridge_test.1q.acceptance.repeat1.Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.7.x86_64.io-github-autotest-qemu.pktgen_perf.q35: PASS (143.45 s)
 (2/2) 82599.bridge_test.1q.acceptance.repeat1.Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.pktgen_perf.q35: PASS (161.66 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2021-03-14T23.41-78dd146/results.html